### ISSUE TITLE
After configuring the Office Online plugin for Nextcloud, I am unable to open docx, xlsx, and pptx files properly.

### ISSUE THREAD

**Issue by a54758398** [nextcloud/officeonline#572]
After configuring the Office Online plugin for Nextcloud, I am unable to open docx, xlsx, and pptx files properly.
![image-2024-7-24_10-28-8](https://github.com/user-attachments/assets/c3b0678b-5b8c-45fa-b30c-5c4ec47bd1eb)
When the Nextcloud Office Online plugin connects to the Microsoft cloud-based Office Online for web test server, the docx, xlsx, and pptx files cannot be opened correctly.
We would like to inquire about what configurations need to be made to ensure proper functionality. (https://ffc-onenote.officeapps.live.com/hosting/discovery)
![image-2024-7-24_11-56-0](https://github.com/user-attachments/assets/a0c527ae-7f8c-4307-80c7-673f4c590c63)
Microsoft cloud-based Office Online for web test server:
![image-2024-7-24_9-55-43](https://github.com/user-attachments/assets/d1eae9aa-a86a-4ac0-9262-4062026e32fd)
Nextcloud version:24.0.3.
Office online app version:2.2.1.
We would like to know if the issue of not being able to open docx, xlsx, and pptx files is related to Microsoft's cloud-based Office Online for web testing server.

**Comment by koelle25** [comment]
I'm not a maintainer here, but did you add your nextcloud URL to the "WOPI domain allow list" mentioned in your screenshot from the Office documentation?
In general I have no idea if this is possible with the Microsoft-Cloud based Office Online. In my case I have an on-premise Office Online running besides Nextcloud, and this is working.

**Comment by koelle25** [comment]
Also: Are you actually a Cloud Storage Partner? As the [note here](https://learn.microsoft.com/en-us/microsoft-365/cloud-storage-partner-program/online/overview) says:
> #### Important
> The Office 365 - Cloud Storage Partner Program is for independent software vendors whose business is cloud storage. It's not open to Office 365 customers directly.

**Comment by juliusknorr** [comment]
Yes, we currently do not support office 365 in the cloud as stated in the readme.

**Comment by a54758398** [comment]
> I'm not a maintainer here, but did you add your nextcloud URL to the "WOPI domain allow list" mentioned in your screenshot from the Office documentation? In general I have no idea if this is possible with the Microsoft-Cloud based Office Online. In my case I have an on-premise Office Online running besides Nextcloud, and this is working.



> Also: Are you actually a Cloud Storage Partner? As the [note here](https://learn.microsoft.com/en-us/microsoft-365/cloud-storage-partner-program/online/overview) says:
> 


Thank you very much for your response.
It was very insightful.

Yes, we have added the Nextcloud URL to the "WOPI domain allow list" for the Microsoft Office Online for Web server.
However, the current issue persists.

Previously, we also built an on-premises Office Online Server (OOS), but we had to discontinue its use due to several issues.
We reached out to Microsoft support regarding issues, such as "[File Save As Error · Issue #268 · nextcloud/officeonline](https://github.com/nextcloud/officeonline/issues/268)" and "Image Attachment Error." 
While Microsoft acknowledged and reproduced these errors, they informed us that there were no plans to update the on-premises Office Online, and as a result, support was discontinued.

Although this product is within the scope of our Microsoft contract, the lack of bug fixes led us to abandon the on-premises Office Online solution.

It's interesting to hear that you've deployed Office Online locally and that it is functioning correctly. 
Did you experience any issues similar to those we encountered?

Additionally, as you pointed in your response, we have obtained certification as a Cloud Storage Partner. 
It seems the issues we faced with Office Online Server (OOS) are resolved in the cloud version, and we are now considering making the switch.

**Comment by a54758398** [comment]
> Yes, we currently do not support office 365 in the cloud as stated in the readme.

Thank you very much for your response.

Understood that only Office Online Server (OOS) is supported, while Office 365 in the cloud is not.
Also reviewed the README, which states: "The cloud-based Office 365 is not supported."
https://github.com/nextcloud/officeonline/blob/main/README.md#server

Have you been able to use Office Online Server (OOS) without encountering any issues?
We have been facing several challenges with Office Online Server (OOS) and are now considering a WOPI-based connection that supports cloud integration.
